PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -4591,66 +4591,76 @@ bool ScintillaEditView::pasteToMultiSelection() const
 
 	// "MSDEVColumnSelect" is column format from Scintilla 
 	CLIPFORMAT cfColumnSelect = static_cast<CLIPFORMAT>(::RegisterClipboardFormat(TEXT("MSDEVColumnSelect")));
-	if (IsClipboardFormatAvailable(cfColumnSelect) && OpenClipboard(NULL))
+	if (::IsClipboardFormatAvailable(cfColumnSelect) && ::OpenClipboard(NULL))
 	{
 		HANDLE clipboardData = ::GetClipboardData(CF_UNICODETEXT);
-		::GlobalSize(clipboardData);
-		LPVOID clipboardDataPtr = ::GlobalLock(clipboardData);
-		if (clipboardDataPtr)
+		if (!clipboardData)
 		{
-			wstring clipboardStr = (const TCHAR*)clipboardDataPtr;
-			::GlobalUnlock(clipboardData);
 			::CloseClipboard();
-
-			vector<wstring> clipboardStrings;
-			stringSplit(clipboardStr, getEOLString(), clipboardStrings);
-			clipboardStrings.erase(clipboardStrings.cend() - 1); // remove the last empty string
-			size_t nbClipboardStr = clipboardStrings.size();
-
-			if (nbSelections >= nbClipboardStr) // enough holes for every insertion, keep holes empty if there are some left
+		}

```

Comment:
Will do here.
Because this PR has started to be bigger, I tried to change as little code as possible. But this place really needs the changes you proposed.